docs/angular/devtools.md

```diff
@@ -0,0 +1,120 @@
+---
+id: devtools
+title: Devtools
+ref: docs/react/devtools.md
+replace: { 'React': 'Angular' }
+---
+
+## Install and Import the Devtools
+
+The devtools are a separate package that you need to install:
+
+```bash
+$ npm i @tanstack/angular-query-devtools-experimental
+# or
+$ pnpm add @tanstack/react-query-devtools-experimental
+# or
+$ yarn add @tanstack/react-query-devtools-experimental
+```
+
+You can import the devtools like this:
+
+```tsx
+import { AngularQueryDevtools } from '@tanstack/angular-query-devtools-experimental'
+```
+
+By default, React Query Devtools are only included in bundles when `process.env.NODE_ENV === 'development'`, so you don't need to worry about excluding them during a production build.
+
+## Floating Mode
+
+Floating Mode will mount the devtools as a fixed, floating element in your app and provide a toggle in the corner of the screen to show and hide the devtools. This toggle state will be stored and remembered in localStorage across reloads.
+
+Place the following code as high in your React app as you can. The closer it is to the root of the page, the better it will work!
```

Comment:
```suggestion
Place the following code as high in your Angular app as you can. The closer it is to the root of the page, the better it will work!
```

Reply:
It may be worth looking into adding first class support, but [toSignal](https://angular.dev/guide/signals/rxjs-interop#tosignal) may be sufficient. What do you think?